specutils/analysis/moment.py

```diff
@@ -21,7 +21,7 @@ def moment(spectrum, regions=None, order=0, axis=-1):
     spectrum : `~specutils.spectra.spectrum1d.Spectrum1D`
         The spectrum object over which the width will be calculated.
 
-    regions: `~specutils.utils.SpectralRegion` or list of `~specutils.utils.SpectralRegion`
+    regions: `~specutils.SpectralRegion` or list of `~specutils.SpectralRegion`
```

Comment:
```suggestion
    regions: `~specutils.spectra.spectral_region.SpectralRegion` or list of `~specutils.spectra.spectral_region.SpectralRegion`
```
looks like these need the full path, even if they are exposed at the top level (cf. Spectrum1D reference above).